src/ai_helps_pwr/models/gpt.py

```diff
@@ -0,0 +1,23 @@
+"""GPT Api connection."""
+import openai
+
+
+class GPT:
+    """Model to conversation."""
+
+    def __init__(
+        self,
+        name: str,
+        gpt_model_name: str,
+    ):
+        """Init."""
+        self.name = name
+        self.gpt_model_name = gpt_model_name
+
+    def __call__(self, prompt, key):
+        """Post prompt to GPT API."""
+        openai.api_key = key
```

Comment:
To mi się wydaje, że powinno być ustawiane raz np. przy konstruowaniu modelu, no bo po co dla każdego prompta ponownie ustawiać to samo api_key. Jak sądzisz?

Reply:
Jak ustawi się to w i init to nie działa, Stąd takie rozwiązanie.